README-vault.md

```diff
@@ -145,6 +145,8 @@ Example playbook to retrieve vault data from a symmetric vault:
       username: admin
       password: SomeVAULTpassword
       state: retrieved
+    no_log: yes
+    register: result
```

Comment:
Wouldn't it be good to have an example how to use the result like we have it for random passwords for user and host?

Reply:
added example.